bots/images/scripts/lib/kubevirt.setup

```diff
@@ -18,7 +18,9 @@ rm -rf ${KUBEVIRT_DIR}
 mkdir -p ${KUBEVIRT_DIR}
 cd ${KUBEVIRT_DIR}
 
-curl -sL https://api.github.com/repos/kubevirt/kubevirt/releases/latest > kubevirt_latest.json
+# HACK latest release v0.7.0 breaks, see https://github.com/cockpit-project/cockpit/pull/9638
+# curl -sL https://api.github.com/repos/kubevirt/kubevirt/releases/latest > kubevirt_latest.json
+curl -sL https://api.github.com/repos/kubevirt/kubevirt/releases/tags/v0.6.2 > kubevirt_latest.json
```

Comment:
I would just have set `KUBEVIRT_VERSION` to that fixed value, but *shrug*, that works too :-)